src/views/v2/month/calendar-body/day/multiday-events/multiday-event.php

```diff
@@ -45,6 +48,7 @@
 	 * The two values might be the same but they will differ for events that last more than one week.
 	 */
 	$classes[] = 'tribe-events-calendar-month__multiday-event--width-' . $event->this_week_duration;
+	$classes[]    = 'tribe-events-calendar-month__multiday-event--display';
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 1 space but found 4 spaces

<sub>hash: db6bd2020cd88746d298999b33d03b88</sub>